src/modules/counter.test.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * @jest-environment jsdom
```

Comment:
- [ ] Please you test are failing, checkout the screenshot below and fix the issue. It will also be nice to update your readme file with explicit steps on how to clone and run your test locally. Also remember to update `package.json file` with the script to run your test(`e.g. "test": "jest"`). Update the script section of your package.json file with the command to run your tests. 

![Screenshot 2023-03-18 163328](https://user-images.githubusercontent.com/65852621/226143677-6770c290-3201-4f1a-a7e2-363c6046b66d.png)